src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -84,6 +95,11 @@
     public final String retentionTimeMinutesStr;
     public final String launchTimeoutSecondsStr;
     public final String bootDiskSizeGbStr;
+    public final boolean windows;
```

Comment:
As mentioned below, consider switching this to an os enum for future flexibility.

Reply:
tracking issue.